packages/elements/src/types.js

```diff
@@ -63,7 +62,9 @@ StoryPropTypes.box = PropTypes.exact({
 
 StoryPropTypes.page = PropTypes.shape({
   id: PropTypes.string.isRequired,
-  animations: PropTypes.arrayOf(PropTypes.shape(AnimationProps)),
+  // Temporary solution for animations. Better would be to move this
+  // prop type to the types package, but really?
+  animations: PropTypes.arrayOf(PropTypes.object),
```

Comment:
As noted, this isn't great, but okay until we convert this package to TypeScript.

Reply:
> Better would be to move this prop type to the types package

Is this referring to just the animation prop type? Or all of `StoryPropTypes` in general?

Technically it would make sense for all of the `StoryPropTypes.*` prop types to live next to their respective TS counterparts, e.g. `StoryPropTypes.page` in the `story` package, `StoryElementPropTypes` in `elements`, and so on.

But yeah I guess with the conversion to TypeScript the prop types usage will slowly fade away anyway (unless we see a lot of value in keeping it around)

Reply:
It's just this single prop type, because the dependency between the animation and the elements package have been reversed. Yes, it would make sense to move them all to the types package, I'll look at that in #12126.